editor/src/components/inspector/sections/component-section/data-picker-popup.tsx

```diff
@@ -70,6 +70,14 @@ function valueToDisplay(option: VariableOption): string {
   }
 }
 
+function isChildrenProp(path: PropertyPath): boolean {
+  return (
+    path.propertyElements.length > 0 &&
```

Comment:
what if the path is longer than 1 element? it feels like we should error out in that case (not necessarily here, but this code reminded me of the problem)

Reply:
fair concern, I'll address this in follow-up PR so the decision around this edge isn't muddled together with the code on this PR